web/src/app/lib/i18n/i18n.service.ts

```diff
@@ -51,7 +51,16 @@ export class I18nService {
   }
 
   public getTranslation(phrase:string, opts?:object):string {
-    return this.data && this.data[phrase] ? this.data[phrase] : phrase
+    return this.data && this.data[phrase] ? this.interpolate(this.data[phrase], opts) : phrase
+  }
+
+  private interpolate(phrase:string, opts?:object): string {
+    const toBeTranslated = phrase.match(/__([^ ]*)__/gm);
+    if (!toBeTranslated) return phrase;
+    for (var i = 0; i < toBeTranslated.length; i++) {
+      phrase = phrase.replace(toBeTranslated[i], opts && opts[toBeTranslated[i].replace(/__/g, "")]);
+    }
+    return phrase;
```

Comment:
adds backwards compatibility with the `__varname__` format of the current locale files